docs/api-reference/xviz-builder.md

```diff
@@ -6,39 +6,35 @@ provides convenient chaining methods to build up XVIZ data in an easy-to-read wa
 ## Example
 
 ```js
-import {XVIZBuilder} from '@xviz/builder';
+import { XVIZBuilder } from '@xviz/builder';
 
 const xvizBuilder = new XVIZBuilder({
   metadata: {} // See XVIZMetadataBuilder for generating metadata object
 });
 
 xvizBuilder
-  .pose({
-    time: 123,
-    latitude: 12.345,
-    longitude: 12.345,
-    altitude: 12.345,
-    roll: 0.123,
-    pitch: 0.123,
-    yaw: 0.123
-  })
+  .pose()
+  .timestamp(123)
+  .mapOrigin(12.345, 12.345, 12.345)
+  .position(123, 123, 123)
+  .orientation(0.123, 0.123, 0.123)
 
+xvizBuilder
   .variable('/velocity')
```

Comment:
Our documentation here is not quite accurate.

'/vehicle/velocity' is supposed to be a `timeSeries` stream.
 
```
xvizBuilder
  .timeSeries('/vehicle/velocity')
```

Here is the schema detail about `TimeSeries` and `Variables`.

[TimeSeries](https://github.com/uber/xviz/blob/master/docs/protocol-schema/core-types.md#time-series-state)
[Variable](https://github.com/uber/xviz/blob/master/docs/protocol-schema/core-types.md#variables)

Reply:
Makes sense, I was confused why variable would require an array anyway. Will fix this once I get back to work tomorrow.